dashboard/app/views/levels/_admin.html.haml

```diff
@@ -84,7 +84,7 @@
       %br
       %button{type: 'button', onclick: 'Blockly.mainBlockSpace.clear()', class: 'btn btn-default btn-sm'}
         Clear workspace
-      - if @level.uses_google_blockly? || params[:blocklyVersion] == 'google' || Experiment.enabled?(experiment_name: 'google_blockly', user: current_user)
+      - unless params[:blocklyVersion] == 'cdo'
```

Comment:
This line determines whether we should add an undo button to the Extra links box or not. Previously, we had to handle several possible paths for determining the Blockly version, but now we can just check the URL flag.